MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/fragment/SortItemsDialogFragment.java

```diff
@@ -0,0 +1,115 @@
+package com.izor066.android.mediatracker.ui.fragment;
+
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+
+import com.izor066.android.mediatracker.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class SortItemsDialogFragment extends DialogFragment implements RadioGroup.OnCheckedChangeListener {
+
+    private RadioButton title;
+    private RadioButton author;
+    private RadioButton added;
+    private RadioGroup sortItems;
+    private OnSortingOptionSelectedListener onSortingOptionSelectedListener;
+    private String currentSortCriteria = "added";
+    private boolean isAscending = true;
+
+
+    public SortItemsDialogFragment() {
+        // Required empty public constructor
+    }
+
+    public static interface OnSortingOptionSelectedListener {
+        public void onTitleSelected();
+        public void onAuthorSelected();
+        public void onAddedSelected();
+    }
+
+    @Override
+    public void onAttach(Activity activity) {
+        super.onAttach(activity);
+
+        try {
+            onSortingOptionSelectedListener = (OnSortingOptionSelectedListener) activity;
+        } catch (ClassCastException e) {
+            throw new ClassCastException(activity.toString()
+                    + " must implement OnSortingOptionSelectedListener");
+        }
+    }
+
+
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        Dialog dialog = super.onCreateDialog(savedInstanceState);
+        dialog.setTitle("Select sorting criteria");
+        return dialog;
+
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_sort_items_dialog, container, false);
+
+        title = (RadioButton) view.findViewById(R.id.radio_title);
+        author = (RadioButton) view.findViewById(R.id.radio_author);
+        added = (RadioButton) view.findViewById(R.id.radio_added);
+
+        currentSortCriteria = getArguments().getString("sort");
```

Comment:
See my `enum` comment. As `enums` are automatically serializable there would be no problem of passing it to this fragment.